core/templates/dev/head/collection_editor/navbar/collection_editor_navbar_directive.html

```diff
@@ -0,0 +1,91 @@
+<script type="text/ng-template" id="inline/collection_editor_navbar_directive">
+  <ul
+    class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak"
+    style="margin-left: 20px;">
+    <!-- Editor -->
+    <li
+      ng-class="{'active': getTabStatuses().active === 'main'">
```

Comment:
I tested these before commit.. Probably I hit delete key in my editor before commit, sorry